File: app/src/main/java/com/pdg/androidbasics/utils/PermissionsManager.java
package com.pdg.androidbasics.utils;

import android.Manifest;
import android.content.Context;
import android.content.pm.PackageManager;
import android.support.v4.content.ContextCompat;

public class PermissionsManager {

    public boolean isPermissionGranted(Context context, String permission) {
        if (ContextCompat.checkSelfPermission(context, permission)!= PackageManager.PERMISSION_GRANTED) {
            return false;
        } return true;
    }

    public void getPermission(){
        return;
    }
}

File: app/src/main/java/com/pdg/androidbasics/utils/PermissionsManager.kt
package com.pdg.androidbasics.utils

import android.Manifest
import android.content.Context
import android.content.pm.PackageManager
import android.support.v4.content.ContextCompat

class PermissionsManager {

    fun isPermissionGranted(context: Context, permission: String): Boolean {
        return if (ContextCompat.checkSelfPermission(context, permission) != PackageManager.PERMISSION_GRANTED) {
            false
        } else true
    }

    fun getPermission() {
        return
    }
}
